concent_api/concent_api/system_check.py

```diff
@@ -40,3 +42,17 @@ def check_settings_concent_features(app_configs, **kwargs):  # pylint: disable=u
             ))
 
     return errors_and_warnings
+
+
+@register()
+def check_settings_storage_cluster_address(app_configs, **kwargs):  # pylint: disable=unused-argument
+    storage_cluster_address = URLValidator(schemes=['http', 'https'])
```

Comment:
This variable does not store cluster address so why is it called `storage_cluster_address`? It's totally misleading. This should be called something like `url_validator`.

Please always call things for what they really are. It makes the code easier to read.